data/data/agent/systemd/units/agent-interactive-console.service

```diff
@@ -7,6 +7,7 @@ Before=network.target network.service agent.service NetworkManager-wait-online.s
 Type=oneshot
 TTYPath=/dev/tty15
 ExecStartPre=/usr/bin/chvt 15
+ExecStartPre=mkdir -p /var/log/agent
 ExecStart="/usr/local/bin/agent-interactive-console.sh"
```

Comment:
Could we add `EnvironmentFile=etc/assisted/agent-installer.env` and then ExecStart the agent-tui binary directly and remove the script?

Reply:
Done